packages/playwright-core/src/server/frames.ts

```diff
@@ -1210,6 +1210,10 @@ export class Frame extends SdkObject {
       // Note: do not perform locator handlers checkpoint to avoid moving the mouse in the middle of a drag operation.
       dom.assertDone(await this._retryWithProgressIfNotConnected(progress, target, options.strict, false /* performActionPreChecks */, async handle => {
         return handle._retryPointerAction(progress, 'move and up', false, async point => {
+          // NOTE: Normal browsers emit usually a lot of dragover/mousemove events during drag'n
+          // drop operations. We want to emit minimal to make the ecosystem work. When using
```

Comment:
instead of handwavy "the ecosystem", let's mention specifically that some implementations, including Angular CDK, rely on this.